src/js/angular/clustermanagement/directives/cluster-configuration.directive.js

```diff
@@ -0,0 +1,61 @@
+import {DELETE_CLUSTER, UPDATE_CLUSTER} from "../events";
+
+const modules = [];
+
+angular
+    .module('graphdb.framework.clustermanagement.directives.cluster-configuration', modules)
+    .directive('clusterConfiguration', ClusterConfiguration);
+
+ClusterConfiguration.$inject = ['$jwtAuth', '$uibModal', '$translate', 'toastr', 'ClusterViewContextService', 'ClusterRestService'];
+
+function ClusterConfiguration($jwtAuth, $uibModal, $translate, toastr, ClusterViewContextService, ClusterRestService) {
+    return {
+        restrict: 'E',
+        templateUrl: 'js/angular/clustermanagement/templates/cluster-configuration.html',
+        scope: {
+            currentNode: '=',
+            clusterModel: '=',
+            clusterConfiguration: '='
+        },
+        link: ($scope) => {
+
+            $scope.closeClusterConfigurationPanel = () => {
+                ClusterViewContextService.hideClusterConfigurationPanel();
+            };
+
+            $scope.isAdmin = () => {
```

Comment:
IMO this can be passed as argument to directive or be calculated once when directive is created. We can't change admin when the directive is displayed.

Reply:
I will check this out